conan/test_package/CMakeLists.txt

```diff
@@ -1,49 +1,71 @@
+if (TARGET userver::ubench)
+    add_executable(${PROJECT_NAME}_ubench test_ubench.cpp)
+    target_link_libraries(${PROJECT_NAME}_ubench ${PROJECT_NAME}_objs userver::ubench)
+    add_google_benchmark_tests(${PROJECT_NAME}_ubench)
+endif()
 
-add_executable(${PROJECT_NAME}_redis test_redis.cpp)
-target_link_libraries(${PROJECT_NAME}_redis ${PROJECT_NAME}_objs userver::redis)
+if (TARGET userver::postgresql)
+    add_executable(${PROJECT_NAME}_postgresql test_postgresql.cpp)
+    target_link_libraries(${PROJECT_NAME}_postgresql ${PROJECT_NAME}_objs userver::postgresql)
+endif()
 
-add_executable(${PROJECT_NAME}_rabbitmq test_rabbitmq.cpp)
-target_link_libraries(${PROJECT_NAME}_rabbitmq ${PROJECT_NAME}_objs userver::rabbitmq)
+if (TARGET userver::redis)
+    add_executable(${PROJECT_NAME}_redis test_redis.cpp)
+    target_link_libraries(${PROJECT_NAME}_redis ${PROJECT_NAME}_objs userver::redis)
+endif()
 
-add_executable(${PROJECT_NAME}_grpc test_grpc.cpp)
-userver_add_grpc_library(${PROJECT_NAME}_proto PROTOS greeter.proto)
-target_link_libraries(${PROJECT_NAME}_grpc ${PROJECT_NAME}_objs userver::grpc ${PROJECT_NAME}_proto)
+if (TARGET userver::rabbitmq)
+    add_executable(${PROJECT_NAME}_rabbitmq test_rabbitmq.cpp)
+    target_link_libraries(${PROJECT_NAME}_rabbitmq ${PROJECT_NAME}_objs userver::rabbitmq)
+endif()
 
-add_executable(${PROJECT_NAME}_mongo test_mongo.cpp)
-target_link_libraries(${PROJECT_NAME}_mongo ${PROJECT_NAME}_objs userver::mongo)
+if (TARGET userver::grpc)
+    add_executable(${PROJECT_NAME}_grpc test_grpc.cpp)
+    userver_add_grpc_library(${PROJECT_NAME}_proto PROTOS greeter.proto)
+    target_link_libraries(${PROJECT_NAME}_grpc ${PROJECT_NAME}_objs userver::grpc ${PROJECT_NAME}_proto)
+endif()
 
-add_executable(${PROJECT_NAME}_clickhouse test_clickhouse.cpp)
-target_link_libraries(${PROJECT_NAME}_clickhouse ${PROJECT_NAME}_objs userver::clickhouse)
+if (TARGET userver::mongo)
+    add_executable(${PROJECT_NAME}_mongo test_mongo.cpp)
+    target_link_libraries(${PROJECT_NAME}_mongo ${PROJECT_NAME}_objs userver::mongo)
+endif()
 
-add_executable(${PROJECT_NAME}_universal test_universal.cpp)
-target_link_libraries(${PROJECT_NAME}_universal ${PROJECT_NAME}_objs userver::universal)
+if (TARGET userver::clickhouse)
+    add_executable(${PROJECT_NAME}_clickhouse test_clickhouse.cpp)
+    target_link_libraries(${PROJECT_NAME}_clickhouse ${PROJECT_NAME}_objs userver::clickhouse)
+endif()
 
-add_executable(${PROJECT_NAME}_kafka test_kafka.cpp)
-target_link_libraries(${PROJECT_NAME}_kafka ${PROJECT_NAME}_objs userver::kafka)
+if (TARGET userver::kafka)
+    add_executable(${PROJECT_NAME}_kafka test_kafka.cpp)
+    target_link_libraries(${PROJECT_NAME}_kafka ${PROJECT_NAME}_objs userver::kafka)
+endif()
+
+add_executable(${PROJECT_NAME}_universal test_universal.cpp)
+target_link_libraries(${PROJECT_NAME}_universal ${PROJECT_NAME}_objs userver::core userver::universal)
```

Comment:
could you raise this two lines above the drives configuration
it is more logical, imho